app/views/principals/rejection_form.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="l-constrained">
-  <section class="l-pre-qualification-questions">
-    <%= link_to t('rejection.try_again'), prequalify_principals_path, class: 'l-pre-qualification-questions__try-again t-back' %>
-
+  <%= form_for @message, url: contact_path, as: :contact, html: { class: 'l-pre-qualification-questions' },
```

Comment:
Whilst we're at it, this should also have a class of `form` to get all the styling